main.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'app'
```

Comment:
![image](https://github.com/Zuwairanajma/school_library/assets/93039359/53d4407f-b83a-4f6c-9fad-c33dcbe0e588)

- [ ] It would be nice if you make your app work as expected because the app is not working then there is no point in testing :thinking: Kindly fix your app so we can make sure that the tests are correctly set because these tests are for future means in future if you make any changes to your code and get any issue then these tests will make your work easy to figure out the codebase where error exists :thinking: